docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+web:
+  build: .
+  ports:
+    - "8888:8888"
+  volumes:
+    - .:/src
```

Comment:
If we're mirror volumes, do we need to copy? I don't think in production we should have this. `docker-compose` supports including other configs and you can do production/dev configs this way.